zulipterminal/ui_tools/views.py

```diff
@@ -1365,6 +1378,11 @@ def __init__(self, controller: Any, stream_id: int) -> None:
                         "Stream email",
                         f"Press {email_keys} to copy Stream email address",
                     ),
+                    ("Stream ID", f"{Stream_id}"),
+                    ("Type of Stream", f"{type_of_stream}"),
+                    ("Posting Policy", f"{posting_policy}"),
+                    ("History Available to Subscribers", f"{availability_of_history}"),
+                    ("User Role", f"{user_role}"),
```

Comment:
We may want to re-order some of the fields here. for e.g. Stream Id and type of stream seems more significant than others so we could have those at the top?

Reply:
Rearranged stream id and type to the top.